WordPress/Tests/XSS/EscapeOutputUnitTest.inc

```diff
@@ -60,4 +60,12 @@ echo $foo ? $foo : 'no foo'; // Bad
 echo empty( $foo ) ? 'no foo' : $foo; // Bad
 echo $foo ? esc_html( $foo ) : 'no foo'; // OK
 
-echo 4; // OK
\ No newline at end of file
+echo 4; // OK
+
+exit( $foo ); // Bad
+exit( esc_html( $foo ) ); // OK
+
+die( $foo ); // Bad
+die( esc_html( $foo ) ); // OK
+
+printf( 'Hello %s', $foo ); // Bad
```

Comment:
@JDGrimes I think `printf` here is a tricky one, and it is discussed in #212. We basically need recognize that what you have here is indeed bad, but whitelist the following as OK:

```php
printf( 'Hello %s', esc_html( $foo ) ); // OK
```

That will require some sophistication, so I think this that `printf` and `vprintf` should be removed from the scope of this PR, unless you want to tackle #212 as well :smile: